contrib/completion/bash/docker

```diff
@@ -5503,6 +5500,18 @@ _docker_compose() {
 	COMPREPLY=( $(compgen -W "${result}" -- "$current") )
 }
 
+_docker_compose() {
```

Comment:
typo: ~~_docker_compose~~ _docker_buildx

Reply:
doh!

Still need to verify if this works as expected as well (didn't try yet TBH, as I ran out of steam 😅)